internal/config/config.go

```diff
@@ -438,7 +438,7 @@ func validConfigKeys() []string {
 
 func stringInStrings(s string, ss []string) bool {
 	for _, v := range ss {
-		if strings.EqualFold(v, s) {
+		if v == s {
```

Comment:
This will enforce case sensitivity when validating keys, which is an effective solution for this issue.  Nice job!

However, `stringInStrings` is also used to [validate the config *values*](https://github.com/newrelic/newrelic-cli/blob/master/internal/config/config.go#L339-L349), so case sensitivity would be enforced when checking a log level the user has chosen.  This is less user friendly - we want them to be able to use "INFO", "info", or "Info" for instance.   

How about introducing a boolean arg similar to [this code](https://github.com/hashicorp/terraform-plugin-sdk/blob/master/helper/validation/strings.go#L132) that would allow turning case sensitivity on or off depending on the use case?  We could use the case sensitive version to validate the keys, and the case insensitive version to validate the values.

We will also need to update the case of the keys in the [`validate` method](https://github.com/newrelic/newrelic-cli/blob/master/internal/config/config.go#L339-L349) after enforcing case sensitivity, as those case statements will stop evaluating to true.